source/github/repo.go

```diff
@@ -16,6 +17,20 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// ConfigBackoff is a wrapper for Config that will retry five times if the function fails to retrieve the yaml/yml file.
+func (c *client) ConfigBackoff(u *library.User, org, name, ref string) (data []byte, err error) {
+	for i := 0; i < 5; i++ {
+		data, err = c.Config(u, org, name, ref)
+		if err != nil {
+			return
+		}
+
+		time.Sleep(5 * time.Second)
```

Comment:
I wonder if waiting `5s` between each API call is what we want here.

I think we could improve this by making it an exponential backoff period between each call 👍 

Something like:

```suggestion
		time.Sleep((i + 1) * time.Second)
```

Reply:
agreed. i think most of the time, a minor delay will probably do the job, so a jump to 5s immediately seems a bit steep.